pages/refed.tsx

```diff
@@ -17,13 +19,38 @@ function Refed() {
                     </Card.Header>
                     <Card.Body>
                         <p>You were invited to use this application because someone thought it would be interesting and helpful to you.</p>
+                        <p>
+                            After you are logged in with Google you can verify your Minecraft account{' '}
+                            <Tooltip
+                                content={
+                                    <span>
+                                        <HelpIcon />
+                                    </span>
+                                }
+                                type="hover"
+                                tooltipContent={
+                                    <p>
+                                        To connect your Minecraft account, search and click yourself in the search bar. Afterwards click "Your? Claim account."
```

Comment:
```suggestion
                                        To connect your Minecraft account, search and click yourself in the search bar. Afterwards click "You? Claim account."
```
thats not the text thats on the player page